provider/README.md

```diff
@@ -336,8 +336,51 @@ When multiple query parameters are specified, they should all apply to the retur
 [Top][toc]
 
 ## Realtime Data
+All providers should provide an endpoint that displays the current status of a providers vehicles on the PROW. Only vehicles that are currently in an available, unavailable, or reserved states should be returned in this payload. Data in this endpoint should generally reconcile with data from the status changes enpdoint.
```

Comment:
In today's MDS call, we talked about marking this /vehicles endpoint as "optional" in 0.4.1 and likely promoting it to "required" in 0.5.0. I'd be curious to see how this could best be expressed in this diff. Right now, this line says "All providers should" provide this endpoint - how should this wording be modified?

Would modifying that portion of the line to read something more like "Providers MAY provide an endpoint that..." be sufficient, or should this documentation also clearly call out the plan to mark this endpoint as required in 0.5.0? I don't have any particular preference re: wording, I just want to make sure that we write this down somewhere so that people who weren't on the call can get the full picture!

Reply:
I think our use of words like MAY/SHOULD and MUST has been fairly consistent and we would continue with that approach here. But you raise a good point that visibility of this changing requirement is important. Maybe it would be useful to add a line to each endpoint description indicating if it is required or not:

```diff
Endpoint: /vehicles
+ Required/Optional: Optional
Method: GET
Schema: vehicles schema
data Payload: { "vehicles": [] }, an array of objects with the following structure
```

Reply:
As discussed on the 2/13 call, we may want to add a note about the future development path. e.g:

     Required/Optional: Optional (may become required in a future major release)